src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -107,21 +91,74 @@ def get_metadata(self) -> dict:
         return metadata
 
     def get_original_timestamps(self) -> np.ndarray:
-        raise NotImplementedError(
-            "Unable to retrieve the original unaltered timestamps for this interface! "
-            "Define the `get_original_timestamps` method for this interface."
-        )
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
+
+    def get_timestamps(self) -> Optional[np.ndarray]:
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
+
+    def align_timestamps(self, aligned_timestamps: List[np.ndarray]):
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
 
-    def get_timestamps(self) -> np.ndarray:
+    def align_starting_time(self, starting_time: float):
         raise NotImplementedError(
-            "Unable to retrieve timestamps for this interface! Define the `get_timestamps` method for this interface."
+            "The VideoInterface operates on a list of file paths; to reduce ambiguity, please choose "
+            "between `align_global_starting_time` (shift starting time of each video by the same value) "
+            "and `align_starting_times` (specify a list of values to use in shifting the starting time for each video)."
         )
 
-    def align_timestamps(self, aligned_timestamps: np.ndarray):
-        raise NotImplementedError(
-            "The protocol for synchronizing the timestamps of this interface has not been specified!"
+    def align_global_starting_time(self, global_starting_time: float, stub_test: bool = False):
```

Comment:
I'd really like to keep the consistent form of the API if we can. I think if a single float is input it's pretty clear that every video should be shifted by that float in seconds, and we can use `align_starting_time` for this without worrying. Cody, what is your concern with doing that? How are you worrying that will be misused? I don't understand why there is a need for `align_global_starting_time`.

Reply:
So a class like this should have two methods named `align_starting_time(starting_time=...)` and `align_starting_times(starting_times=...)` and the presence of pluralization is sufficient to avoid any confusion about what each method does? And will never lead to any minor mistakes?

My 2 cents, a 1 character difference is hard to notice visually; I can foresee a less verbose approach as being much harder to debug than having an explicit method name that makes it clear how it applies to a scalar rather than a vector

Reply:
I agree, separating these long function names by a single character could cause confusion. What if we instead change `align_starting_times` to something else, like `align_each_starting_time`. That would allow us to keep a consistent API for `align_starting_time` and would make it easy to differentiate between the functions. Does that satisfy your concerns?

Reply:
It's better, but the word 'each' isn't very descriptive...

What about defining the base method to be `align_global_starting_time` since that is truly what it does in _all_ interfaces, not just the ones with multiple segments or objects?

And then for the recording and sorting interfaces that have multiple segments, have it be called `align_segment_starting_times`?

And then for the audio and video interfaces that have multiple files (that may not be 'segments', but rather a contiguous stream getting concatenated smoothly) call it `align_file_starting_times`? Or I guess we could still get away with calling it `align_segment_starting_times` in that case, they just happen to be segments that are evenly partitioned from the full stream...

Reply:
I'd be happy with `align_starting_time` and `align_segment_starting_times`. I'm not a big fan of the word "global" here, because it makes me think that this might somehow have to do with a global variable, or it would somehow be applied to all interfaces.

Reply:
`align_segment_starting_times` it is then

Adjusted it in the audio PR, might add some follow-ups to improve naming consistency for multi-segment recording/sortings once everything is in `main`